tests/src/core/testqgsadjacenttiles.cpp

```diff
@@ -0,0 +1,171 @@
+/***************************************************************************
+     testadjacenttiles.cpp
```

Comment:
Considering wonder-sks comment that QgsVectorLayerRenderer is an internal class, I'm replacing QgsVectorLayerRenderer with a class in the interface (QgsMapRendererSequentialJob ?)